libraries/exoplayer_rtsp/src/main/java/androidx/media3/exoplayer/rtsp/RtpPayloadFormat.java

```diff
@@ -71,6 +75,10 @@ public static String getMimeTypeFromRtpMediaType(String mediaType) {
         return MimeTypes.VIDEO_H265;
       case RTP_MEDIA_MPEG4_GENERIC:
         return MimeTypes.AUDIO_AAC;
+      case RTP_MEDIA_AMR:
+        return MimeTypes.AUDIO_AMR_NB;
+      case RTP_MEDIA_AMR_WB:
+        return MimeTypes.AUDIO_AMR_WB;
```

Comment:
Upon setting the mimeType as AUDIO_AMR, the player fails in selectTracks. The reason for this failure is: AUDIO_AMR is treated as an unsupported mime type and therefore result.second is set to null here:
https://github.com/androidx/media/blob/main/libraries/exoplayer/src/main/java/androidx/media3/exoplayer/trackselection/MappingTrackSelector.java#L422
 
Regarding the second change, i.e. mapping AMR_WB to audio/3gpp, since AUDIO_AMR_NB creates an AMR_NB decoder and AUDIO_AMR_WB creates AMR_WB decoder we see the following playback error:
`MediaCodecDecoderException: Decoder failed: c2.android.amrnb.decoder`